src/library/scala/Unit.scala

```diff
@@ -31,6 +31,9 @@ final abstract class Unit private extends AnyVal {
 object Unit extends AnyValCompanion {
 
   /** Transform a value type into a boxed reference type.
+   *
+   *  This method is not intended for use in source code.
+   *  The runtime representation of this value is platform-specific.
    *
    *  @param  x   the Unit to be boxed
    *  @return     a scala.runtime.BoxedUnit offering `x` as its underlying value.
```

Comment:
This is a pretty un-platform-specific return value.

Reply:
OK, changed to `pretty platform-specific`. Or should it be `pretty-platform-specific`?

I just learned, actually, that it should not have a hyphen: `it is platform specific`, vs, `it a platform-specific feature`. Someone in the crossword comments brought the issue to my attention after I joked about it.